master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -106,10 +106,16 @@ spec:
         k8s-app: calico-node
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        scheduler.alpha.kubernetes.io/tolerations: |
-          [{"key": "dedicated", "value": "master", "effect": "NoSchedule" },
-           {"key":"CriticalAddonsOnly", "operator":"Exists"}]
     spec:
+       tolerations:
```

Comment:
This same change needs to be made below to the `calico-kube-controllers`, it still has the old toleration style.

Reply:
I think with the calico-kube-controllers we only need the `CriticalAddonsOnly` toleration right? If a node is flagged to keep workloads off of it (e.g. master or special nodes) I don't think we want to force kube-controllers onto it?

Reply:
I don't know, we could maybe drop the NoExecute, but the previous did have a NoSchedule bit so that should maybe stay. @caseydavenport WDYT?

Reply:
kube-controllers shouldn't tolerate _all_ taints, but it should be able to run on the master, and it should be marked as critical.

Unlike `calico/node`, I don't think it should tolerate running on any `NoSchedule` / `NoExecute` nodes.

Reply:
I've included

```yaml
- key: node-role.kubernetes.io/master
          effect: NoSchedule
```

in the controllers part of the manifest tolerations.